templates/ehub.json

```diff
@@ -60,6 +60,20 @@
             "metadata": {
                 "description": "Please use the actual consumer group if using existing event hub."
             }
+        },
+        "Event Hub Filter Json": {
+            "type": "String",
+            "defaultValue": "",
+            "metadata": {
+                "description": "Optional. Please enter type of filter in JSON format"
```

Comment:
please add a simple example.
Also mention that only messages which contain the specified property will be collected.

Reply:
Done